deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -41,11 +51,14 @@ public String inputDirectory() {
     public boolean trigger(CodeGenContext context) throws CodeGenException {
         final Path outDir = context.outDir();
         final Path openApiDir = context.inputDir();
+        final List<String> ignoredFiles = context.config()
+                .getOptionalValues("quarkus.openapi-generator.codegen.ignore", String.class).orElse(List.of());
 
         if (Files.isDirectory(openApiDir)) {
             try (Stream<Path> openApiFilesPaths = Files.walk(openApiDir)) {
                 openApiFilesPaths
                         .filter(Files::isRegularFile)
+                        .filter(not(path -> ignoredFiles.contains(path.getFileName().toString())))
```

Comment:
Should now we move this filter after the Map::toString
I have to confess there is to many streaming here for my taste ;), but it we are gonna play streaming, then lets play it smartly, below this filter is a string conversion and another filter, so lets put it there
I would also argue that this filter should be the last one of the chain, because it is the slower one

Reply:
No, because `Path#toString` returns the full path and I need only the filename. Note that I use `path.getFileName().toString()` in the second filter.

Reply:
Done.